dynaconf/loaders/vault_loader.py

```diff
@@ -130,3 +130,17 @@ def write(obj, data=None, **kwargs):
     path = "/".join([obj.VAULT_PATH_FOR_DYNACONF, obj.current_env.lower()])
     client.secrets.kv.create_or_update_secret(path, secret=data)
     load(obj)
+
+
+def load_keys(obj, path):
```

Comment:
What do you think about naming it `list_envs` ?
```suggestion
def list_envs(obj, path):
```

In that way we can implement the same thing to all the loaders, so we make it uniform, for example, in a future PR we can add `yaml_loader.list_envs` and this will return the list of all the first level keys in the yaml file defined by `path` argument.

Reply:
I like it. I will add that shortly.